packages/core/src/components/forms/numericStepper.tsx

```diff
@@ -0,0 +1,358 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as classNames from "classnames";
+import * as PureRender from "pure-render-decorator";
+import * as React from "react";
+
+import { AbstractComponent, Classes, Keys } from "../../common";
+import * as Errors from "../../common/errors";
+import { HTMLInputProps, IIntentProps, IProps, removeNonHTMLProps } from "../../common/props";
+
+import { Button } from "../button/buttons";
+import { InputGroup } from "./inputGroup";
+
+export enum NumericStepperButtonPosition {
+    NONE,
+    LEFT,
+    RIGHT,
+    SPLIT,
+}
+
+export interface INumericStepperProps extends IIntentProps, IProps {
+
+    /**
+     * The button configuration with respect to the input field
+     * @default "right"
+     */
+    buttonPosition?: NumericStepperButtonPosition;
```

Comment:
They're a bit noisy in there while not interactive. Since we control the component, I figured we could hide them. I'm pretty sure left position would be useful for RTL for instance